### ISSUE TITLE
Link from Geome project page to AmphibianDisease.org project page

### ISSUE THREAD

**Issue by mkoo** [BNHM/AmphibiaWebDiseasePortal#35]
Another one probably better on the Geome issue list:

Can we have a link on the GEOME Project page that links back to the Amphibian disease project page? This is related to DOI #12 links.
 I am assuming we will have semi-permalinks like https://amphibiandisease.org/projects/?id=286
which would be nice to have on https://geome-db.org/workbench/project-overview?projectId=286

Maybe another "Shareable URLs"? 

Right now, there are no links returning user to AmphibianDisease.org from Geome

**Comment by jdeck88** [comment]
The data guid does this, i think?  see:

https://geome-db.org/workbench/project-overview?projectId=223

the data guid is listed as:

https://n2t.net/ark:/21547/APc2

that is a bit cryptic though (and not clickable i see!)

**Comment by mkoo** [comment]
Well not yet... the ark link should work once #12 is complete, right?
It's just not obvious so would be nice to have some customized url options on Geome

**Comment by mkoo** [comment]
Can this issue be transferred to Geome-UI issue list @jdeck88 ? [#540](https://github.com/biocodellc/geome-ui/issues/540)
We would like to have a link from Geome project pages to the ADP project page. Came up in user feedback
